pkg/storaged/storage-controls.jsx

```diff
@@ -215,8 +215,8 @@ export const StorageUsageBar = ({ stats, critical, block, offset, total, small }
         </div>);
 };
 
-export const StorageMenuItem = ({ onClick, onlyNarrow, children }) => (
-    <DropdownItem className={onlyNarrow ? "show-only-when-narrow" : null}
+export const StorageMenuItem = ({ onClick, onlyNarrow, danger, children }) => (
+    <DropdownItem className={(onlyNarrow ? "show-only-when-narrow" : null) + " " + (danger ? "delete-resource-red" : null)}
```

Comment:
Because javascript is weird this is not correct entiretly :)

Reply:
`"" + null` will result to "null"

For this reason you need to replace null with '' and  you can also remove the middle + " " + spacer, but just adding a space in the " delete-resource-red"

Reply:
Hello @KKoukiou!

I've removed the merge commit and changed the null condition, let me know if I should make any more changes, thanks :)